src/homepageExperience/components/steps/arduino/InstallDependencies.tsx

```diff
@@ -1,115 +1,28 @@
 
 export const InstallDependencies: FC = () => {
-  const dispatch = useDispatch()
-
-  useEffect(() => {
-    dispatch(getBuckets())
-  }, [])
-
-  const headingWithMargin = {marginTop: '48px', marginBottom: '0px'}
-
-  const logCopyCodeSnippet = OS => {
-    event(`firstMile.cliWizard.installDependencies${OS}.code.copied`)
-  }
-
-  type CurrentOSSelection = 'Linux' | 'Mac' | 'Windows'
-  const [currentSelection, setCurrentSelection] = useState<CurrentOSSelection>(
-    'Mac'
-  )
-
-  useEffect(() => {
-    const fireKeyboardCopyEvent = event => {
-      if (
-        keyboardCopyTriggered(event) &&
-        userSelection().includes('brew install')
-      ) {
-        logCopyCodeSnippet('Mac')
-      }
-      if (
-        keyboardCopyTriggered(event) &&
-        (userSelection().includes('Expand-Archive') ||
-          userSelection().includes('mv'))
-      ) {
-        logCopyCodeSnippet('Windows')
-      }
-      if (
-        keyboardCopyTriggered(event) &&
-        (userSelection().includes('wget') ||
-          userSelection().includes('tar') ||
-          userSelection().includes('sudo'))
-      ) {
-        logCopyCodeSnippet('Linux')
-      }
-    }
-    document.addEventListener('keydown', fireKeyboardCopyEvent)
-    return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
-  }, [])
-
   return (
     <>
       <h1>Install Dependencies</h1>
-      <ButtonGroup orientation={Orientation.Horizontal}>
-        <Button
-          text="mac OS"
-          color={
-            currentSelection === 'Mac'
-              ? ComponentColor.Primary
-              : ComponentColor.Default
-          }
-          onClick={() => {
-            setCurrentSelection('Mac')
-          }}
-        />
-        <Button
-          text="windows"
-          color={
-            currentSelection === 'Windows'
-              ? ComponentColor.Primary
-              : ComponentColor.Default
-          }
-          onClick={() => {
-            setCurrentSelection('Windows')
-          }}
-        />
-        <Button
-          text="linux"
-          color={
-            currentSelection === 'Linux'
-              ? ComponentColor.Primary
-              : ComponentColor.Default
-          }
-          onClick={() => {
-            setCurrentSelection('Linux')
-          }}
-        />
-      </ButtonGroup>
-      {currentSelection === 'Mac' && (
-        <>
-          <h2 style={headingWithMargin}>Use Homebrew</h2>
-        </>
-      )}
-      {currentSelection === 'Windows' && (
-        <>
-          <h2 style={headingWithMargin}>Download the CLI package</h2>
-        </>
-      )}
-      {currentSelection === 'Linux' && (
-        <>
-          <h2 style={headingWithMargin}>Download from the command line</h2>
-        </>
-      )}
+      <p>
+        You can install the required InfluxDB Client for Arduino library from
+        the library manager
+      </p>
+      <ol style={{fontSize: '16px', fontWeight: 'normal'}}>
```

Comment:
[this same style](https://github.com/influxdata/ui/blob/master/src/homepageExperience/components/steps/arduino/PrepareIde.tsx#L4) is defined in the `PrepareIde` file. Let's follow up on this and create a class in `HomepageExperience.scss` and apply the class to the `ol` elements in this file and `PrepareIde`

Reply:
Will follow up.